grade.go

```diff
@@ -372,12 +386,10 @@ func main() {
 		var result util.SubmissionResult
 		results.Results[dir] = &result
 		results.Order = append(results.Order, dir)
-
 		result.Student = dir
 		// find hydratedStudent information from EUID (dirname)
 		hydratedStudent := db.GetStudentByEuid(supabase, dir)
 
-		// if student doesn't exist, commit to db
```

Comment:
this comment was moved further down- noted in its new location, check rebase for other similar issues

Reply:
Will definitely check and see if there was any code that was moved out of order